File: CafeEase/CafeEase.Model/Table.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CafeEase.Model
{
    public class Table
    {
        public int Id { get; set; }
        public int Number { get; set; }
        public int Capacity { get; set; }
        public bool IsOccupied { get; set; }
    }
}

File: CafeEase/UI/cafeease_desktop/lib/models/table.dart
import 'package:json_annotation/json_annotation.dart';

part 'table.g.dart';

@JsonSerializable()
class Table {
  int? id;
  int? number;
  int? capacity;
  bool? isOccupied;

Table({
  this.id,
  required this.number,
  required this.capacity,
  required this.isOccupied,
});

  factory Table.fromJson(Map<String, dynamic> json) =>
      _$TableFromJson(json);

  Map<String, dynamic> toJson() => _$TableToJson(this);
}
